types/datatables.net/index.d.ts

```diff
@@ -1604,7 +1604,9 @@ declare namespace DataTables {
     }
 
     interface FunctionColumnRender {
-        (data: any, t: string, row: any, meta: CellMetaSettings): void;
+        (data: any, type: 'filter' | 'display' | 'type' | 'sort', row: any, meta: CellMetaSettings): any;
+        (data: any, type: undefined, row: any, meta: CellMetaSettings): any;
+        (data: any, type: any, row: any, meta: CellMetaSettings): any;
```

Comment:
why not put this into one signature like the test. That is, `'filter' | ... | undefined | any`?